cmd/scollector/collectors/redis_unix.go

```diff
@@ -236,6 +236,33 @@ func c_redis() (opentsdb.MultiDataPoint, error) {
 				Add(&md, "redis."+sp[0], status(sp[1]), tags, metadata.Unknown, metadata.None, "")
 				continue
 			}
+			if strings.HasPrefix(sp[0], "cmdstat_") {
+				cmdStats := strings.Split(sp[1], ",")
+				if len(cmdStats) < 3 {
+					continue
+				}
+				cmdStatsCalls := strings.Split(cmdStats[0], "=")
+				cmdStatsUsec := strings.Split(cmdStats[1], "=")
+				var cmdStatsMsec, cmdStatsMsecPc float64
+				microsec, err := strconv.ParseFloat(cmdStatsUsec[1], 64)
```

Comment:
Need to check length here to avoid possible panic, don't know that cmdStatusUsec is of len > 1